### PR TITLE
image-path-fixes-inside-adoc-file

### PR DESCRIPTION
**Pull request by pankajp-ts**
Tickets - 
https://thoughtspot.atlassian.net/browse/SCAL-77550
https://thoughtspot.atlassian.net/browse/SCAL-77763

### CODE REVIEW

**Review by ayonghosh — APPROVED** [review]
Should we also do an automated find and replace on these?

File: docs/src/components/Page404/index.scss [hunk 1]
```diff
@@ -1,25 +1,25 @@
-@import '../../static/styles/variables.scss';
+@import '../../assets/styles/variables.scss';
 
 .pageStyles {
-  color: #232129;
-  padding: 96px;
-  font-family: -apple-system, Roboto, sans-serif, serif,
+    color: #232129;
```
Comment (ayonghosh) [inline docs/src/components/Page404/index.scss:4]: Use a variable for this colour

File: docs/src/components/Page404/index.scss [hunk 2]
```diff
@@ -1,25 +1,25 @@
...
+    padding: 96px;
+    font-family: -apple-system, Roboto, sans-serif, serif;
```
Comment (ayonghosh) [inline docs/src/components/Page404/index.scss:6]: Shouldn't this be Optima Prime?

File: docs/src/components/Page404/index.scss [hunk 3]
```diff
@@ -1,25 +1,25 @@
...
-  color: #8A6534;
-  padding: 4;
-  background-color: #FFF4DB;
-  font-size: $font-size-404;
-  border-radius: 4;
-}
\ No newline at end of file
+    color: #8a6534;
+    padding: 4;
+    background-color: #fff4db;
```
Comment (ayonghosh) [inline docs/src/components/Page404/index.scss:22]: Colour variable

File: gatsby-config.js [hunk 4]
```diff
@@ -1,11 +1,50 @@
+const asciidoc = require(`asciidoctor`)();
+const config = require('./docs/src/configs/doc-configs');
+
+class CustomTSConverter {
```
Comment (ayonghosh) [inline gatsby-config.js:4]: What's the TS supposed to mean here?

Comment (pankajp-ts) [reply gatsby-config.js:4]: 'Type String'

File: docs/src/components/Page404/index.scss (hunk 1 above)
Comment (pankajp-ts) [reply docs/src/components/Page404/index.scss:4]: done

File: docs/src/components/Page404/index.scss (hunk 2 above)
Comment (pankajp-ts) [reply docs/src/components/Page404/index.scss:6]: done

File: docs/src/components/Page404/index.scss (hunk 3 above)
Comment (pankajp-ts) [reply docs/src/components/Page404/index.scss:22]: done

File: gatsby-config.js (hunk 4 above)
Comment (pankajp-ts) [reply gatsby-config.js:4]: renamed to `CustomDocConverter`

File: docs/src/components/Page404/index.scss [hunk 5]
```diff
@@ -1,25 +1,25 @@
-@import '../../static/styles/variables.scss';
+@import '../../assets/styles/variables.scss';
 
 .pageStyles {
-  color: #232129;
-  padding: 96px;
-  font-family: -apple-system, Roboto, sans-serif, serif,
+    color: $primarycolor;
+    padding: 96px;
+    font-family: 'Optimo-Plain';
```
Comment (ayonghosh) [inline docs/src/components/Page404/index.scss:6]: Add the sans-serif fallback

**Review by ayonghosh — APPROVED** [review]
LGTM after adding the fallback font

File: docs/src/components/Page404/index.scss (hunk 5 above)
Comment (pankajp-ts) [reply docs/src/components/Page404/index.scss]: done

**Pull request merged**